content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md

```diff
@@ -0,0 +1,54 @@
+# Add Dependencies or Outputs to a Stage
+
+There are situations where we have executed a stage (either by writing
```

Comment:
In this case title sounds too general for the `dvc commit` scenario. It sounds like all we need to describe in this section is `open dvc.yaml - edit` or execute `dvc run` again. If it happened that files are already in place use --no-exec + `dvc commit`

Reply:
In both cases (either editing `dvc.yaml` or using `--no-exec` with `run`), `dvc commit` is required. So that's why it's a separate scenario in `dvc commit`.

Reply:
> too general for the dvc commit scenario... all we need to describe in this section is...

@shcheklein do you mean in https://github.com/iterative/dvc.org/pull/1914/files#diff-b8f371cb91c101e733f37e95de023afe203b77b8a0443b3d453a76739b675852R43-R48 above?

The scenario in `commit` is "we have previously executed a stage and later notice that some files/directories are missing from `deps` or `outs` in `dvc.yaml`" which seems to me exactly like what this doc addresses.

> Cc @imhardikj that would be the whole text in it's most compressed form BTW.

Or are you saying that the long text and example in this how-to is unnecessary and could be compressed into a single paragraph (thus maybe not even needing a whole how-to page)?

Thanks

Reply:
My point is that when I read `Add Dependencies or Outputs to a Stage` I would expect to a much simpler scenario first - `Open your file, add it manually, save file. Run repro or commit in some cases.` - that kind of a thing. Not one of the very specific sub-scenarios from the `dvc commit`.

Reply:
Gotcha Ivan. You're right. We should simplify the intro to this doc and separate that from the example below @imhardikj. I'll resolve this conversation and reopen in the exact places...